### PR TITLE
Clean old schemas

### PR DESCRIPTION
**Pull request by osheroff**
@vanchi-zendesk our database is growing toooooo big

### CODE REVIEW

File: src/main/java/com/zendesk/maxwell/schema/SchemaStore.java [hunk 1]
```diff
@@ -297,9 +309,23 @@ public void destroy() throws SQLException {
...
+		ResultSet rs = connection.createStatement().executeQuery("select id from maxwell.schemas where id = " + schema_id);
+		return rs.next();
+	}
+
 	public BinlogPosition getBinlogPosition() {
 		return this.position;
 	}
 
+	private void deleteOldSchemas(Long currentSchemaId) throws SQLException {
+		Long toDelete = currentSchemaId - SCHEMAS_TO_KEEP;
```
Comment (vanchi-zendesk) [inline src/main/java/com/zendesk/maxwell/schema/SchemaStore.java]: is this a valid assumption? (That ids are incrementing properly). Also why are we not deleting everything older?

Comment (vanchi-zendesk) [inline src/main/java/com/zendesk/maxwell/schema/SchemaStore.java]: guessing by the name, I think we missed a toDelete-- in there.

Comment (osheroff) [inline src/main/java/com/zendesk/maxwell/schema/SchemaStore.java]: derp.  yeah.  missed `--`

Comment (osheroff) [inline src/main/java/com/zendesk/maxwell/schema/SchemaStore.java]: and yeah, I think we can assume auto-increment works

**Comment by osheroff** [comment]
fixed

**Comment by vanchi-zendesk** [comment]
:+1: :)

**Pull request merged**